translation/transport.go

```diff
@@ -279,7 +272,7 @@ func requestAddPayload(m map[string]string, input io.Reader) (p []byte, err erro
 
 	wdmp.Table = table
 
-	payload, err := ioutil.ReadAll(input)
+	payload, _ := ioutil.ReadAll(input)
```

Comment:
Why don't we want to check this error?

Reply:
added a check for this. Initially I thought any error would have been addressed in line 281.